### ISSUE TITLE
Foliate does not open the file second time unless cache is deleted

### ISSUE THREAD

**Issue by raulod** [johnfactotum/foliate#1415]
**Describe the bug**
Foliate opens the file fine. However, when trying to open it the second time, it just hangs. 

However if I delete the cache folder and try again, it opens fine for one more time. And the next time, same issue repeats. 

**To Reproduce**
Follow above steps

**Version:**
 - Foliate version: foliate-1.5.3-3.el8.noarch
 - OS/Distribution and version:  RHEL 8.10
 - Desktop environment: [e.g. GNOME 3.36]
 - Installation method: yum

**Additional context**

**Comment by johnfactotum** [comment]
Version 1.5 is ancient and not supported. Consider using the latest release, v3.1.

**Comment by raulod** [comment]
1). Official version in RHEL 8 is 1.5 and there is no way to update it. There is no attempt from you to provide packages 
2) I tried downloading source and your use of meson >= 0.59 was another roadblock
3) and ninja

Then I came across this https://ludditus.com/2021/11/29/linux-apps-the-failure-of-foliate/ and he recommend koodo and it turned out to be a better choice

https://www.koodoreader.com/en 

I wish I could have used foliate but there are too many reasons to move away from it.
